src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsInstanceMetadataTagsEnum.java

```diff
@@ -0,0 +1,38 @@
+package com.spotinst.sdkjava.enums.aws.elastigroup;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public enum AwsInstanceMetadataTagsEnum {
+
+    ENABLED("enabled"),
```

Comment:
this should be in Enabled and Disabled according to openAPI document

Reply:
Document is to be corrected. The working values are "enabled" and "disabled". No changed needed.